scripts/automation/trex_control_plane/stl/console/trex_tui.py

```diff
@@ -13,6 +13,8 @@
 from trex_stl_lib.utils.text_opts import *
 from trex_stl_lib.utils import text_tables
 from trex_stl_lib import trex_stl_stats
+import trex_root_path
```

Comment:
We changed the model to a self contained model.
files from stl/* cannot point outside.

i will move your file filter.py to under trex_stl_lib.utils